### PR TITLE
[WIP] Add Sony Xperia L aka taoshan

### PR DESCRIPTION
**Pull request by jonnius**

### CODE REVIEW

File: manifests/sony_taoshan.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<manifest>
+    <remote name="jonnius"
+        fetch="https://github.com/jonnius"
+        revision="halium.7.1" />
+
+	<project path="device/sony/taoshan" name="android_device_sony_taoshan" remote="jonnius" />
+	<project path="kernel/sony/msm8930" name="android_kernel_sony_msm8930" remote="jonnius" />
+	<project path="vendor/sony" name="proprietary_vendor_sony" revision="cm-14.1" remote="them" />
```
Comment (lnjX) [inline manifests/sony_taoshan.xml]: You can also remove the revision attribute since it's also cm-14.1 for TheMuppets.

**Comment by lnjX** [comment]
Isn't this already compiling? Then we can merge this 😀

**Comment by jonnius** [comment]
Yes, it is compiling and installing, but not working.

**Comment by lnjX** [comment]
I know but to be added here, it doesnt has to completely work, it's just important that nothing gets lost :) And you can create a new pull request anytime!

**Pull request merged**